src/vsr/superblock.zig

```diff
@@ -32,6 +32,9 @@ pub const Quorums = @import("superblock_quorums.zig").QuorumsType(.{
 
 pub const SuperBlockVersion: u16 = 0;
 
+const vsr_headers_reserved_size = constants.sector_size -
+    ((constants.view_change_headers_max * @sizeOf(vsr.Header)) % constants.sector_size);
+
 // Fields are aligned to work as an extern or packed struct.
 pub const SuperBlockSector = extern struct {
```

Comment:
I'm not sure whether we should rename `SuperBlockSector` now that it is (at minimum) 2 sectors in size.

Reply:
:+1: I'll rename in a separate PR.